packages/amplify-graphql-maps-to-transformer/src/__tests__/__integ__/with-many-to-many.test.ts

```diff
@@ -52,7 +52,7 @@ const transformSchema = (schema: string): DeploymentResources => {
 describe('mapsTo with manyToMany', () => {
   it('creates resources with original GSIs and field names', () => {
     const out = transformSchema(manyToManyMapped);
-    expect(out.stacks.EmployeeTask!.Resources!.EmployeeTaskTable.Properties.GlobalSecondaryIndexes).toMatchSnapshot();
+    expect(out.stacks.EmployeeTask!.Resources!.EmployeeTaskTable09DF6270.Properties.GlobalSecondaryIndexes).toMatchSnapshot();
     const outSchema = parse(out.schema);
```

Comment:
This is likely coming from model transformer. Logged follow up https://app.asana.com/0/1202389680978480/1203146351085134/f .